src/Microsoft.AspNetCore.Mvc.TagHelpers/PartialTagHelper.cs

```diff
@@ -22,6 +23,7 @@ public class PartialTagHelper : TagHelper
     {
         private const string ForAttributeName = "for";
         private const string ModelAttributeName = "model";
+        private const string FallbackAttributeName = "fallback";
```

Comment:
@DamianEdwards would like this to be called `fallback-name` so that it's consistent with `name`.